time_series_models/processes.py

```diff
@@ -511,16 +515,16 @@ def make_preprocessor(self, lags=None, **kwargs):
                 make_pv_pipeline(self.hrrr_fetcher, self._pv_builder),
                 [0],
             ),
-            (
-                "correlated_generation",
-                autoregressive_features_pipeline(
-                    self.range_fetcher,
-                    lags,
-                    self.tstep,
-                    **kwargs,
-                ),
-                [0],
-            ),
+            # (
+            #     "correlated_generation",
+            #     autoregressive_features_pipeline(
+            #         self.range_fetcher,
+            #         lags,
+            #         self.tstep,
+            #         **kwargs,
+            #     ),
+            #     [0],
+            # ),
```

Comment:
add a "# TODO <reason why this is commented out>" ?

Reply:
Moar TODO!